### ISSUE TITLE
Incorrect keyboard handling for creator type menu

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4248]
When the item type menu has keyboard focus, you can press up/down to open the menu, and you can type for immediate keyboard navigation to a menu option. On the creator-type menu, neither of those works — up/down just scrolls the pane.

(Testing on macOS. I think Windows is slightly different and actually changes the selection on up/down-arrow.)

**Comment by dstillman** [comment]
This worked in Z6.

**Comment by abaevbog** [comment]
That's caused by the creator type menu not being a real `menulist` like itemType menu but rather a `div` from where we open the popup ourselves. For me, arrowUp/down did open up creator menu in z6 but typing when the creator is focused does not change the selection (e.g. clicking "e" does not change the creator type to Editor). In any case, it should be consistent across all menu dropdowns that we have.

I think ideally we would make creator type menu a `menulist` which should give us all the desired keypress handling out of the box. The issue with that is this would require some additional refactoring of itemBox, since `<menupopup id="creator-type-menu" position="after_start"/>` is used in quite a few places. So it might be easier to just add the extra keydown handling to the creator rows.

**Comment by dstillman** [comment]
I don't think we should try to implement this ourselves.

Why did up/down work in Z6? Were we handling that explicitly? This is not a big deal, so we could do just that for now and swap in a proper menu later…

**Comment by abaevbog** [comment]
What do you think about this? It seems to preserve the original behavior from zotero 6 of opening the creator type menu on `arrowUp` or `arrowDown`. 

> Testing on macOS. I think Windows is slightly different and actually changes the selection on up/down-arrow.

Even though on windows these buttons do change the selection for, say, item type menu, they still just opened the creator type menu for me. So this change will be consistent with what we had before.

And then I will try to refactor the itemBox (maybe after the custom section row API is done to avoid conflicts) to have a proper `menulist` instead of a `div` for the menu.
